### ISSUE TITLE
Add option to resume training

### ISSUE THREAD

**Issue by avisingh599** [avisingh599/visual-qa#4]

**Comment by avisingh599** [comment]
Also add option to specify number of epochs, model saving frequency etc. as input arguments

**Comment by anantzoid** [comment]
Hi,
Should I add the `resume training` feature?
I guess the option to save at intervals is already present (via model_save_interval).

**Comment by avisingh599** [comment]
Yes, if you add the feature, feel free to submit a pull request. But please
note that I am not maintaining the codebase actively as of now, so you
should make extensive checks on your end.

Thanks,
Avi
